src/base/_component.scss

```diff
@@ -37,6 +37,17 @@ button {
 	}
 }
 
+/*
+Panels
+ */
+.panel, .well {
+	&:not(:has(.panel, .well)):has(.stretched-link) {
+		&:hover {
+			box-shadow: 1px 5px 7px rgba(0,0,0,.15);
+		}
+	}
```

Comment:
This don't work in FF only in Chrome. Do we have a little FF equivalent? Maybe a lighter effect?

Reply:
There is no way to achieve this styling without ":has()" selector. If the styling HAS to work on Firefox, I'll have to put this PR on hold until Firefox is up to date with every other browser.